dev-libs/libtpms/files/libtpms-0.10.0-tpm2-Add-padding-to-OBJECT-for-32bit-targets.patch

```diff
@@ -0,0 +1,30 @@
+From 095a085b6f447551110e09b2013c6a21c2a29f2d Mon Sep 17 00:00:00 2001
```

Comment:
Please always include bug links/commit links/... at the top of patches.

Reply:
A link to the Gentoo bug has been added.